### DISCUSSION TITLE
Can I probably "hasura-auth" or "hasura-storage" install by docker-compose?

### DISCUSSION THREAD

**Discussion by theMeysam** [nhost/nhost#162] [Q&A] [answered]
Hi dear friends,

After Stopping releasing for Hasura Backend Plus.
May I know, how can I install "hasura-auth" or "hasura-storage" via docker-compose on ubuntu server?

Best regards;

**Comment by plmercereau** [comment, marked as answer]
Hello @theMeysam,
Yes, you can. [Here is an example to run hasura-auth](https://github.com/nhost/hasura-auth/blob/main/docker-compose-example.yaml).
A full-range hasura-auth + hasura-storage example is not ready yet, but we'll write once a bit later - we would like to update hasura-auth/storage before doing so
